Added title support for the menu

Tijgerd opened the pull request:
Added another menu-method with support for a title and sub-title
Using this method instead of the other will remove the "hint" at the bottom of the menu.
I created this method as a first step to simplify the file-selection menu
(which contains code that needs to be in another .cpp file, like Adafruit_Arcada_Alerts.cpp).

Tijgerd committed:
Added support for menu titles

and additional sub-title

Tijgerd committed:
Added support for displaying a title in the menu

And an optional sub-title

Tijgerd committed:
Added class-name before class-method

silly me

Tijgerd committed:
Removed default value of the sub-string

Tijgerd committed:
Removed default values for menu in .cpp

Tijgerd committed:
Update Adafruit_Arcada_Alerts.cpp

Tijgerd committed:
Default value of menu_subtitle to ""

Default value of menu_subtitle to "", so its optional

Tijgerd commented:
@ladyada I know bumping is not really a nice thing to do for a PR.. but... (small) bump :)

ladyada commented:
hiya we had to refactor this library. can you show me what the new menus look like, a picture?

ladyada commented:
can u have just one function for menus, you can add the menutitle at the end of the param list, so it defaults to something (rather than 2 nearly identical functions)

Tijgerd commented:
Just ordered the PyGamer, will fix this as soon as I can after receiving the package :)

Tijgerd committed:
Merged the 2 menu-functions

Tijgerd committed:
Update Adafruit_Arcada.h

Tijgerd committed:
Merged the 2 menu functions

Tijgerd committed:
Touchscreen to button double check fix

Removed double check for touchscreen-to-button LEFT button

Tijgerd committed:
Fixed title and subtitle

I tested the added title and sub-title support!
In the previous commit, the menu-items weren't moved down enough (x-axis)
Will upload photos (or EXTREME-awesome-paint-pro images) to the PR

Tijgerd commented:
![image](https://user-images.githubusercontent.com/17268988/61864660-13cbcf80-aed2-11e9-9bc4-febf09cb4814.png)
I tested this and it works like a charm :)
Keep in mind that I only tested this with a max of 4 menu-items, don't know what the menu will do when the menu is too big.
But I think that is something for a new PR

Tijgerd committed:
Update titles x-position

ladyada commented:
ok will review this next time we're working on this library!
